lib-injection/sources/min_compatible_versions.csv

```diff
@@ -67,6 +67,7 @@ flask,~=0.12.0
 flask-caching,~=1.10.0
 flask-openapi3,0
 gevent,~=20.12.0
+git+https://github.com/DataDog/dd-trace-api-py,0

```

Comment:
should we omit this? I don't think anyone is going to install this way via git, definitely won't be the preferred way in the future, right?

do we require this to exist for SSI to keep working when this package is installed this way?